src/cmds/openapi.js

```diff
@@ -105,8 +105,8 @@ exports.run = async function (opts) {
     }
 
     let bundledSpec;
-    const oas = new OASNormalize(specPath, { enablePaths: true });
-    await oas.validate().catch(err => {
+    const oas = new OASNormalize(specPath, { colorizeErrors: true, enablePaths: true });
+    await oas.validate(false, { validate: { colorizeErrors: true } }).catch(err => {
```

Comment:
This option controls the pretty errors that'll be shown. If `colorizeErrors` is false (it's false by default) those errors will still look nice they just won't have colors or bold text.